tools/javadoc/javadoc.bzl

```diff
@@ -19,6 +19,10 @@ def _android_jar(android_api_level):
         return None
     return Label("@androidsdk//:platforms/android-%s/android.jar" % android_api_level)
 
+def _endswith_java(item):
```

Comment:
What other files do you expect in `srcs`?

Reply:
In my case, my code generator produces both `.java` and `.xml`. One could imagine another tree artifact consisting of Java sources from `.../src/main/java/**/*.java` as well as any misc. resources from `.../src/main/resources/**`.

The `javadoc` utility will choke if any input file doesn't have a `.java` extension, so either I could (within reason) allow for some flexibility in this rule, or I could push the responsibility of filtering onto the users. Since it's cheap/easy, I opted w/ the former.